parsers/nmparser/get_model_output.go

```diff
@@ -67,48 +68,137 @@ func GetModelOutput(filePath string, verbose bool, noExt bool, noGrd bool, noCov
 		}
 	}
 
+	epsCount := 0
+	for i := range results.ParameterStructures.Sigma {
+		if results.ParameterStructures.Sigma[i] > 0 {
+			epsCount++
+		}
+	}
+	etaCount := 0
+	for i := range results.ParameterStructures.Omega {
+		if results.ParameterStructures.Omega[i] > 0 {
```

Comment:
the parameter structure should be nonzero for non-diagonal elements of omega though if the user uses an $OMEGA BLOCK(X) form though? I don't think this logic is correct. The etas should correspond to the diagonal elements of the omega matrix

Reply:
We are looking for the number of etas here to possibly pad ParameterStructures.Omega[].
If this is a valid use case, we can't useParameterStructures.Omega[] to get the number of etas.